data/json/furniture_and_terrain/furniture_vitrified.json

```
@@ -0,0 +1,880 @@
+    "bash": {
+      "str_min": 8,
+      "str_max": 30,
+      "sound": "glass crunching!",
+      "sound_fail": "clink!",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_cupboard",
+    "name": "black glass cupboard",
+    "symbol": "#",
+    "description": "A black glass cupboard.  The doors are fused shut, but it presumably contains glasses.",
+    "color": "dark_gray",
+    "move_cost_mod": 1,
+    "coverage": 55,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "MOUNTABLE", "FLAT_SURF" ],
+    "connect_groups": "COUNTER",
+    "connects_to": "COUNTER",
+    "rotates_to": "INDOORFLOOR",
+    "deconstruct": { "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ] },
+    "bash": {
+      "str_min": 8,
+      "str_max": 30,
+      "sound": "glass crunching!",
+      "sound_fail": "clink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    },
+    "examine_action": "workbench",
+    "workbench": { "multiplier": 1.1, "mass": 2000, "volume": "75L" }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_rack_coat",
+    "name": "black glass coat rack",
+    "description": "A tall glass pole with a set of hooks, on which rigid, shapeless masses are draped.",
+    "symbol": "Y",
+    "color": "dark_gray",
+    "move_cost_mod": -1,
+    "coverage": 30,
+    "required_str": 4,
+    "flags": [ "TRANSPARENT", "BLOCKSDOOR", "MOUNTABLE" ],
+    "deconstruct": { "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ] },
+    "max_volume": "30 L",
+    "bash": {
+      "str_min": 6,
+      "str_max": 30,
+      "sound": "glass shattering!",
+      "sound_fail": "clink.",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_planter",
+    "name": "black glass garden",
+    "description": "A planter of dark glass.  Sharp, twisting glass sprouts emerge from the black soil within.",
+    "symbol": "^",
+    "color": "dark_gray",
+    "move_cost_mod": 1,
+    "required_str": 10,
+    "flags": [ "TRANSPARENT", "PLANTABLE", "FLAT", "MOUNTABLE", "SHARP", "PAINFUL" ],
+    "bash": {
+      "str_min": 6,
+      "str_max": 14,
+      "sound": "crunch.",
+      "sound_fail": "plink!",
+      "items": [ { "item": "black_glass_shard", "count": [ 40, 80 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_vitrified_flower",
+    "name": "black glass flower",
+    "description": "A delicate little thing; a wispy stem slightly bowed under the weight of translucent petals.  The tiny barbs that line it are almost impossible to see, until it crumbles in your fingers.",
+    "looks_like": "f_flower_tulip",
+    "symbol": "f",
+    "color": "light_gray",
+    "move_cost_mod": 0,
+    "required_str": -1,
+    "flags": [ "TRANSPARENT", "TINY", "NOCOLLIDE", "FLOWER", "SHARP", "PAINFUL" ],
```

```suggestion
    "flags": [ "TRANSPARENT", "TINY", "FLOWER", "SHARP", "PAINFUL" ],
```
Seems like if it's sharp it would shatter from a car driving over it.